### ISSUE TITLE
UnicodeDecodeError

### ISSUE THREAD

**Issue by KevinLee752** [YoYo000/MVSNet#49]
Hi！ I'm trying to run the test code under Python3. Then I got this error when it run to "image = scipy.misc.imread(image_file, mode='RGB')" in the MVSGenerator class: 

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte.

Can I please get some advice from you (or any others)?  Thank you.

**Comment by YoYo000** [comment]
There might be some problems using python3. Can python2 successfully pass this part?

You can try using cv2.imread() to replace scipy.misc.imread()

**Comment by KevinLee752** [comment]
Yes, it can. The test code works fine with python2 on my machine. Thanks for your advice. But sorry to bother you again, Does the input image size for test.py has to be 1152x864 for '3DCNN' or 1600x1200 for GRU‘？What if I want to test other sizes of images?
